packages/node/src/models/proposed-app-instance-info.ts

```diff
@@ -74,6 +81,49 @@ export class ProposedAppInstanceInfo implements AppInstanceInfo {
     this.proposedToIdentifier = proposeParams.proposedToIdentifier;
     this.initialState = proposeParams.initialState;
     this.intermediaries = proposeParams.intermediaries;
+    this.id = overrideId || this.getIdentityHashFor(channel!);
+  }
+
+  // TODO: Note the construction of this is duplicated from the machine
+  getIdentityHashFor(stateChannel: StateChannel) {
+    const proposedAppInterface: AppInterface = {
+      addr: this.appId,
+      ...this.abiEncodings
+    };
+
+    const proposedTerms: Terms = {
+      assetType: this.asset.assetType,
+      // FIXME: xuanji
+      limit: Zero, // bigNumberify(this.myDeposit).add(bigNumberify(this.peerDeposit)),
```

Comment:
missed this on my last review, but the commented code is wrong. limit = 0 is expected